.github/workflows/build_test_ci.yml

```
@@ -12,6 +12,10 @@ on:
     branches: [ main ]
   pull_request:
     branches: [ main ]
+  schedule:
+    # Run daily tests to catch issues that could arise on a specific day
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '45 4 * * *'
```

YAMLlint (1.37.1)

[warning] too many spaces after colon

(colons)